code/datums/keybinding/living.dm

```diff
@@ -59,3 +59,18 @@
 	var/mob/living/L = user.mob
 	L.end_look_down()
 	return TRUE
+
+/datum/keybinding/living/rest
+	hotkey_keys = list("U")
```

Comment:
```suggestion
	hotkey_keys = list("ShiftR")
```
Bit of bikeshedding, but the keybind system supports key combos, and `shift` + `R` was a very common macro for resting back when I played on CM.
It's an optional change, though, if you think `U` is better then go for it.
Either way I'll merge this when you reply. Good change!

Reply:
Shift +R could be an issue if people want to do other combinations with shift, and also throw/catch items. I think U is alright.

Reply:
Why would it it be an issue? You press `shift` and then you press `R` and you rest, without triggering throw/catch or other shift combinations.
Regardless, merging it as promised.